tests/openqa/webui/test_results.pm

```diff
@@ -40,7 +40,7 @@ sub run {
     wait_still_screen;
 
     # while job not finished
-    for (1 .. 5) {
+    for (1 .. 15) {
```

Comment:
Wouldn't it be better to simply do a curl call to check the status of the job, and once the job is finished, do do that assert and click?...

Reply:
Apart from the fact that I changed the wrong code (the right code is a few lines down), I prefer the ui method as it not only verifies that the UI is working and provies more information on the test module results but it also prevents the screensaver from starting.